status-server/src/public/ClientPage.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Example Side by Side client</title>
+  <link rel="stylesheet" href="./style/client.css">
+</head>
+<body>
+  <h1>Example status client</h1>
+
+  <section class="container" id="visualization">
+    <div id="band">
+      <canvas id="led0" class="led" width=40 height=40></canvas>
+      <canvas id="led1" class="led" width=40 height=40></canvas>
+      <canvas id="led2" class="led" width=40 height=40></canvas>
+      <canvas id="led3" class="led" width=40 height=40></canvas>
+      <canvas id="led4" class="led" width=40 height=40></canvas>
+    </div>
+  </section>
+
+  <section class="container">
+    <input type="button" value="Start"
+	         onclick="togglePolling(this);" />
+  </section>
+  <section>
+
+    <iframe src="../status0.html" frameborder="0" scrolling="no" height="512" align="left"> </iframe>
```

Comment:
We need a way to pass through an identity to each instance. Perhaps a querystring parameter on either the html page like example.html?id=status0 .. lets talk about the experience we want here